src/common/models/dimension/dimension.ts

```diff
@@ -179,7 +204,13 @@ export class Dimension implements Instance<DimensionValue, DimensionJS> {
       this.kind === other.kind &&
       this.url === other.url &&
       immutableArraysEqual(this.granularities, other.granularities) &&
-      granularityEquals(this.bucketedBy, other.bucketedBy);
+      granularityEquals(this.bucketedBy, other.bucketedBy) &&
+      this.bucketingStrategy === other.bucketingStrategy &&
+      this.sortStrategy === other.sortStrategy;
+  }
+
+  public shouldBucket(): boolean {
```

Comment:
Okay so I'm gonna be nitpicky here but I'm somehow not super fan of the name `shouldBucket` as it implies it's not up to the caller. `canBucket` is more API-like IMHO... Thoughts ?